helm/charts/determined/templates/master-config.yaml

```diff
@@ -16,10 +16,10 @@ data:
       bucket: {{ required "A valid Values.checkpointStorage.bucket entry is required!" .Values.checkpointStorage.bucket }}
       prefix: {{ .Values.checkpointStorage.prefix | quote }}
       {{- else if eq .Values.checkpointStorage.type "s3" }}
-      bucket: {{ required "A valid Values.checkpointStorage.bucket entry is required!" .Values.checkpointStorage.bucket }}
-      access_key: {{ .Values.checkpointStorage.accessKey | quote }}
-      secret_key: {{ .Values.checkpointStorage.secretKey | quote }}
-      endpoint_url: {{ .Values.checkpointStorage.endpointUrl | quote }}
+      bucket: {{ .Values.s3.bucket }}
+      access_key: {{ .Values.s3.accessKey }}
+      secret_key: {{ .Values.s3.secretKey }}
+      endpoint_url: https://s3.{{ .Values.region | lower }}.coreweave.com
```

Comment:
is this is being set here why do we need this value in values.schema.json? we can remove it from the schema otherwise.

Reply:
I removed it. Maybe refresh?